assets/alertmanager/alertmanager.yaml

```diff
@@ -51,7 +51,7 @@ spec:
       name: web
     resources:
       requests:
-        cpu: 10m
+        cpu: 1m
```

Comment:
wow now thats drastic :D

Reply:
this is the alertmanager-proxy (oauth-proxy) sidecar. In the e2e run I analzyed ~1millicore CPU:
![image](https://user-images.githubusercontent.com/375856/76526046-3add0e80-646d-11ea-82cb-cc4f9cb54556.png)

Reply:
Unfortunately anything less than 5m has no effect, the scheduler has an implicit floor at 5m.

Reply:
Let me double check this, we have a minimum floor for memory.

Reply:
Nm, we don't.  This should be fine.